ar5523: use ANSI C, instead of C++, comments.

Changes-licensed-under: ISC
Signed-off-by: Stefan Lippers-Hollmann <s.l-h@gmx.de>

--- a/drivers/net/wireless/ath/ar5523/ar5523.c
+++ b/drivers/net/wireless/ath/ar5523/ar5523.c
@@ -69,7 +69,7 @@ enum {
 	usb_rcvbulkpipe((dev), AR5523_DATA_RX_PIPE)
 
 enum {
-	// XXX: msec or HZ?
+	/* XXX: msec or HZ? */
 	AR5523_DATA_TIMEOUT		= 10000,
 	AR5523_CMD_TIMEOUT		= 1000,
 };
@@ -594,7 +594,7 @@ static void ar5523_data_rx_cb(struct urb
 
 	error = usb_submit_urb(urb, GFP_ATOMIC);
 	if (error) {
-		// XXX: handle
+		/* XXX: handle */
 		ar5523_err(ar, "error %d when resubmitting rx data urb\n",
 			       error);
 	}
@@ -1104,7 +1104,7 @@ static void ar5523_cmd_rx_cb(struct urb
  resubmit:
 	error = usb_submit_urb(urb, GFP_ATOMIC);
 	if (error) {
-		// XXX: handle
+		/* XXX: handle */
 		ar5523_err(ar, "error %d when resubmitting rx urb\n", error);
 	}
 }
@@ -1363,7 +1363,7 @@ static int ar5523_load_firmware(struct u
 	const struct firmware *fw;
 	void *fwbuf;
 	int len, offset;
-	int foolen; // XXX(hch): handle short transfers
+	int foolen; /* XXX(hch): handle short transfers */
 	int error = -ENXIO;
 	
 	if (request_firmware(&fw, AR5523_FIRMWARE_FILE, &dev->dev)) {
